pkg/manager/http.go

```diff
@@ -113,6 +115,13 @@ func (serv *HTTPServer) httpAction(w http.ResponseWriter, r *http.Request) {
 	switch r.FormValue("action") {
 	case "toggle-expert":
 		serv.expertMode = !serv.expertMode
+	case "pause-unpause":
```

Comment:
```
...?action=toggle-expert
...?action=pause-unpause
```

Instead you can:
```
...?toggle=expert
...?toggle=pause
```
```

Reply:
Done.
At some point we may have non-toggle actions, but I guess we can do toggle for now.